src/signalr/azext_signalr/custom.py

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from azext_signalr.signalr.models import (ResourceSku, SignalRCreateOrUpdateProperties, SignalRCreateParameters)
+
+
+def signalr_create(client, signalr_name, resource_group_name, sku, unit_count, location=None, tags=None):
+    sku = ResourceSku(name=sku, capacity=unit_count)
+    properties = SignalRCreateOrUpdateProperties(hostNamePrefix=signalr_name)
+
+    parameter = SignalRCreateParameters(tags=tags,
+                                        sku=sku,
+                                        properties=properties,
+                                        location=location)
+
+    return client.create_or_update(resource_group_name, signalr_name, parameter)
+
+
+def signalr_delete(client, signalr_name, resource_group_name):
+    return client.delete(resource_group_name, signalr_name)
+
+
+def signalr_list(client, resource_group_name=None):
+    if resource_group_name is None:
```

Comment:
Can be changed to `if not resource_group_name`. [Truthiness in Python](https://www.udacity.com/wiki/cs258/truthiness-in-python)